addons/mail/static/tests/qunit_suite_tests/components/emojis_text_field_tests.js

```diff
@@ -1,40 +1,67 @@
 /**@odoo-module **/
 
 import { getFixture } from "@web/../tests/helpers/utils";
-import { makeView, setupViewRegistries } from "@web/../tests/views/helpers";
+import { start, startServer } from '@mail/../tests/helpers/test_utils';
+import { addFakeModel } from '@bus/../tests/helpers/model_definitions_helpers';
+
+import { testEmojiButton,
+         testEmojiButtonHidden,
+         testEmojiButtonVisible,
+        } from "./emojis_field_common_tests";
 
 QUnit.module("Field text emojis", (hooks) => {
     let target = undefined;
-    let serverData = undefined;
+
+    addFakeModel('fields.text.emojis.user', {
+        foo: {type: 'char', onChange: "1"},
+    });
+    const views = {
+        'fields.text.emojis.user,false,form': `
+            <form>
+                <field name="foo" widget="text_emojis"/>
+            </form>
+        `,
+    };
+    const openTestView = async (readonly = false, mockRPC = null) => {
+        const pyEnv = await startServer();
+        const recordId = pyEnv['fields.text.emojis.user'].create({ display_name: 'test record', foo: 'test' });
+        const startServerArgs = {serverData: { views }};
+        if (mockRPC) {
```

Comment:
I don't think you ever pass "mockRPC", so it could be removed, unless I missed it.